### PR TITLE
[docs] supply/upload_to_play_store - update the quick-start requirement to setup supply correctly

### PR DESCRIPTION
**Pull request by renatobenks**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Motivation and Context

According to https://github.com/fastlane/fastlane/issues/14686 which was closed by inactivity, we should update the supply documentation to have the missing step to setup supply correctly and connect it with PlayStore without having errors.

### Description

Added to the quick-start section on upload to PlayStore documentation what is needed to do before trying to use supply.

**There are other places still missing this described step?**

By the way, there's not another way to upload a build to PlayStore on `supply init`? it wouldn't be possible at all?

### CODE REVIEW

**Comment by renatobenks** [comment]
@googlebot I signed it!

File: fastlane/lib/fastlane/actions/docs/upload_to_play_store.md [hunk 1]
```diff
@@ -49,6 +49,8 @@ The previous p12 configuration is still currently supported.
 
 ## Quick Start
 
+> Before use _supply_ correctly to connect with PlayStore, you'll need setup your app manually first, uploading an only build to PlayStore, at least. Take a look into [fastane/fastlane#14686](https://github.com/fastlane/fastlane/issues/14686) to have a better understanding
```
Comment (rogerluan) [inline fastlane/lib/fastlane/actions/docs/upload_to_play_store.md:52]:
```suggestion
> Before using _supply_ to connect to Google Play Store, you'll need to set up your app manually first by uploading at least one build to Google Play Store. See [fastane/fastlane#14686](https://github.com/fastlane/fastlane/issues/14686) for more info.
```

**Review by rogerluan — CHANGES_REQUESTED** [review]
Mere grammar & style review - I have no understanding of the issue itself

**Comment by renatobenks** [comment]
there's anything else missing to merge it? it's just to additionally have the described the extra step required to use `supply`

**Review by joshdholtz — APPROVED** [review]
This is a great addition! Thanks so much for this ❤

**Pull request merged**